power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -908,6 +1121,7 @@ class TapPositionOptimizerImpl<std::tuple<TransformerTypes...>, StateCalculator,
     Calculator calculate_;
     StateUpdater update_;
     OptimizerStrategy strategy_;
+    // TapChangerBinarySearch use_binary_search_;
```

Comment:
THis got left out

Reply:
Yup, intentional for now. It is the enum to replace the boolean. We still have to make a team descision on how, as it affects interface.